openxr/src/generated.rs

```diff
@@ -1262,16 +1276,7 @@ impl ExtensionSet {
         }
     }
     pub(crate) fn from_properties(properties: &[sys::ExtensionProperties]) -> Self {
```

Comment:
Is retaining this separate entry point useful, or should we migrate to calling the `From` impl directly?